test/specs/utils/tablify.js

```diff
@@ -33,40 +33,32 @@ describe('tablify util', () => {
 	Given('a non-empty object', given.aNonEmptyObject, () => {
 		When('the object is tablified', when.theObjectIsTablified, () => {
 			Then(
-				'the returned table should have a head with the object’s keys',
-				then.theReturnedTableShouldHaveAHeadWithTheObjectKeys,
-			);
-			Then(
-				'the returned table should have a row with the object’s values',
-				then.theReturnedTableShouldHaveARowWithTheObjectValues,
+				'the returned table should have a row with the object’s key values',
+				then.theReturnedTableShouldHaveARowWithTheObjectKeyValues,
 			);
 		});
 	});
 	Given('a nested object', given.aNestedObject, () => {
 		When('the object is tablified', when.theObjectIsTablified, () => {
 			Then(
-				'the returned table should have a head with the object’s nested keys',
-				then.theReturnedTableShouldHaveAHeadWithTheObjectNestedKeys,
-			);
-			Then(
-				'the returned table should have a row with the object’s nested values',
-				then.theReturnedTableShouldHaveAHeadWithTheObjectNestedValues,
+				'the returned table should have a row with the object key and stringified nested values',
+				then.theReturnedTableShouldHaveARowWithTheObjectKeyAndStringifiedNestedValues,
 			);
 		});
 	});
 	Given('a deeply nested object', given.aDeeplyNestedObject, () => {
 		When('the object is tablified', when.theObjectIsTablified, () => {
 			Then(
-				'it should throw print error "Output cannot be displayed in table format: Maximum object depth of 3 was exceeded. Consider using JSON output format."',
-				then.itShouldThrowPrintError,
+				'the returned table should have a row with the object key and stringified nested values',
```

Comment:
What about arrays e.g. of votes? This one only has 4 and is already off the screen.
```
lisky> get transaction 3511360482635196713
╔════════════════════╤══════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╗
║ id                 │ 3511360482635196713                                                                                                                                                                                                                                                                      ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ height             │ 340572                                                                                                                                                                                                                                                                                   ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ blockId            │ 5452350734325765704                                                                                                                                                                                                                                                                      ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ type               │ 3                                                                                                                                                                                                                                                                                        ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ timestamp          │ 63398593                                                                                                                                                                                                                                                                                 ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ senderPublicKey    │ 34682d411cdccb22f7e5f1158279cec04a43caa1ed41c2a511f2369709fcd6be                                                                                                                                                                                                                         ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ senderId           │ 5057926541223936699L                                                                                                                                                                                                                                                                     ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ recipientId        │ 5057926541223936699L                                                                                                                                                                                                                                                                     ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ recipientPublicKey │ 34682d411cdccb22f7e5f1158279cec04a43caa1ed41c2a511f2369709fcd6be                                                                                                                                                                                                                         ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ amount             │ 0                                                                                                                                                                                                                                                                                        ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ fee                │ 100000000                                                                                                                                                                                                                                                                                ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ signature          │ 31890206cc2166e2d2a8c8ba56b4c4f383f8e6adac555ee2d7b5a92075349c8c71aeef683d30c46197410dc8ac6e2ec5a1748c16cdae6dc3783426d859193501                                                                                                                                                         ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ signatures         │                                                                                                                                                                                                                                                                                          ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ confirmations      │ 789                                                                                                                                                                                                                                                                                      ║
╟────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╢
║ asset              │ votes: ["+ef63d95c454a52d91d05a9bc072a7350e55157974afe5ae66ad3cbc27fa25360","+2a0183a7343a7bc29c84bc37a19ed149c478b82189225b47795b873a9716c569","-374ce08c19f30551c1485e1213b10658646a7ed69779c446ffba23c367b404e5","-31cc9655335ba9cf01b8393311bb12bff8ea2a2c219e10d819734ce95e5c15f1"] ║
╚════════════════════╧══════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╝
```

Reply:
votes...!!!